sale_milestone_profile_invoicing/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,8 @@
+This module compares budget sold for milestones product with the time spent by various employee profiles.
```

Comment:
Non-blocking. But for person who don't have much background about timesheet, I don't quite get.
If you could explain a bit more on why and how for this module, would be perfect.